app/assets/stylesheets/mortgage_calculator/layouts/_sticky.scss

```diff
@@ -44,4 +44,44 @@
   .borrowing-range {
     display: none;
   }
+
```

Comment:
Hey @munckymagik I'm not sure why l- prefixes aren't being used, this tool may pre-date the use of the prefixes across the other tools.  I think this will need another task to bring this tool inline with the others in terms on naming